aim/web/ui_v2/src/types/components/LineChart/LineChart.d.ts

```diff
@@ -1,4 +1,5 @@
 import { CurveEnum } from 'utils/d3';
+import HighlightEnum from '../../../components/HighlightModesPopover/HighlightEnum';
```

Comment:
Can we import modules with absolute path to avoid long names ?

Reply:
of course , it's automated imports but i will be more attentive about that , thanks